wp-content/themes/dxw-security-2017/templates/partials/global-footer.php

```diff
@@ -57,11 +57,11 @@
             <div class="footer-social">
                 <ul>
                     <li>
-                        <a href="https://twitter.com/dxw" class="twitter">
+                        <a href="https://bsky.app/profile/dxw.bsky.social" class="bsky">
                             <!--[if gte IE 9]><!-->
-                            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 144 144"><title>Twitter</title><path fill="#eeeeee" d="M107,54V56.4c0,23.85-18.13,51.35-51.3,51.35-10.18,0-19.77-2.23-27.71-7.35a28.5,28.5,0,0,0,4.3.3c8.42,0,16.29-3.65,22.39-8.55a18,18,0,0,1-16.8-12.55,17.74,17.74,0,0,0,3.35.34A18.05,18.05,0,0,0,46,79.3,18,18,0,0,1,31.57,61.6v-.21a18.1,18.1,0,0,0,8.16,2.23,18,18,0,0,1-8.08-15A17.2,17.2,0,0,1,34.1,39.9,51.31,51.31,0,0,0,71.27,58.42a18.81,18.81,0,0,1-.39-4.17,18,18,0,0,1,18-18,18.68,18.68,0,0,1,13.28,5.67,36.24,36.24,0,0,0,11.39-4.38,17.9,17.9,0,0,1-7.95,10A35.21,35.21,0,0,0,116,44.67,36.57,36.57,0,0,1,107,54Z"/></svg>
+                            <svg xmlns="http://www.w3.org/2000/svg" viewBox="-21 -28 144 128"><title>Bluesky</title><path fill="#eeeeee" d="M 70.796875 50.65625 C 70.226562 50.585938 69.636719 50.519531 69.0625 50.433594 C 69.652344 50.5 70.226562 50.585938 70.796875 50.65625 Z M 50 39.035156 C 45.46875 30.324219 33.140625 14.082031 21.683594 6.070312 C 10.695312 -1.613281 6.511719 -0.289062 3.75 0.949219 C 0.574219 2.375 0 7.207031 0 10.042969 C 0 12.878906 1.578125 33.347656 2.605469 36.769531 C 5.988281 48.058594 18.074219 51.875 29.203125 50.636719 C 29.773438 50.550781 30.347656 50.484375 30.9375 50.398438 C 30.363281 50.484375 29.792969 50.570312 29.203125 50.636719 C 12.898438 53.042969 -1.578125 58.921875 17.414062 79.839844 C 38.296875 101.253906 46.023438 75.25 50 62.066406 C 53.976562 75.25 58.542969 100.308594 82.222656 79.839844 C 100 62.066406 87.101562 53.027344 70.796875 50.636719 C 70.226562 50.570312 69.636719 50.5 69.0625 50.414062 C 69.652344 50.484375 70.226562 50.570312 70.796875 50.636719 C 81.925781 51.859375 93.992188 48.042969 97.394531 36.769531 C 98.421875 33.347656 100 12.894531 100 10.042969 C 100 7.1875 99.425781 2.359375 96.25 0.949219 C 93.507812 -0.269531 89.304688 -1.613281 78.332031 6.070312 C 66.859375 14.082031 54.53125 30.324219 50 39.035156 Z M 50 39.035156"/></svg>
```

Comment:
There are possibly better ways to do this than messing with the viewport?

Reply:
There's not much you can do with positioning once the svg has been created - it's all set in the co-ordinates. So unless we want to recreate it at the centre of a larger canvas as the others seem to be, I think it's an ok way to go

Reply:
However, something about it seems to have offset the list item slightly on smaller screens

Reply:
It gets fixed if you set the ul to be display flex and justify content center